Documentation/KnownIssues/wpf-pma-window-drag.md

```diff
@@ -0,0 +1,7 @@
+# Per-Monitor DPI aware WPF applications positions mouse cursor incorrectly when dragging between monitors
+
+## Symptoms
+A user running a WPF application that enables the Per-Monitor DPI awareness feature introduced in .NET Framework 4.7.2 may notice that the mouse cursor is in a different location relative to the window after dragging the window from one monitor to another.
+
+## Cause
+This issue is caused by a calculation error when resizing and placing the window after switching DPI.
```

Comment:
Is a hotfix going to be made available? If so, it should be mentioned, as it is in #990, for example.

Reply:
@vatsan-madhavan what's the plan for this bug?

Reply:
We don’t plan on fixing this in .net framework. It will be addressed in .net core.